src/main/java/org/jenkinsci/maven/plugins/hpi/MavenArtifact.java

```diff
@@ -55,14 +56,14 @@ public MavenArtifact(
         this.project = Objects.requireNonNull(project);
     }
 
-    public MavenProject resolvePom() throws ProjectBuildingException {
-        ProjectBuildingRequest buildingRequest =
-                new DefaultProjectBuildingRequest(session.getProjectBuildingRequest());
-        buildingRequest.setProcessPlugins(false); // improve performance
-        buildingRequest.setRemoteRepositories(project.getRemoteArtifactRepositories());
-        buildingRequest.setValidationLevel(ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL);
-        return builder.build(artifact, buildingRequest).getProject();
-    }
+    // public MavenProject resolvePom() throws ProjectBuildingException {
+    //     ProjectBuildingRequest buildingRequest =
+    //             new DefaultProjectBuildingRequest(session.getProjectBuildingRequest());
+    //     buildingRequest.setProcessPlugins(false); // improve performance
+    //     buildingRequest.setRemoteRepositories(project.getRemoteArtifactRepositories());
+    //     buildingRequest.setValidationLevel(ModelBuildingRequest.VALIDATION_LEVEL_MINIMAL);
+    //     return builder.build(artifact, buildingRequest).getProject();
+    // }
```

Comment:
This changes the public API of the class.  Are you confident that there are no consumers of that API outside this repository?
```suggestion
```